components/Carousel/Carousel.js

```diff
@@ -1,8 +1,43 @@
 class Carousel {
-
+    constructor(carousel){
+        this.carousel=carousel; 
+        this.left = left; 
+        this.right=right;
+    
+        
+        this.right.addEventListener('click', () => 
+        images.style.display="block"
+        );
+    }
+    
+    //methods
+    nextImage(){
+        const image = document.querySelectorAll('.carousel img');
+        
+        this.images.style.display="block";
+        
+        // .forEach( function(img){
+        //     images.style.display= "block";
+        // });
+           
+    }
 }
 
-let carousel = document.querySelector();
+const carousel = document.querySelector('.carousel');
```

Comment:
I'd move all the variables to the top here just so we're sure they're all there before we use them.